packages/commonwealth/client/scripts/navigation/CustomDomainRoutes.tsx

```diff
@@ -28,9 +29,15 @@ const FinishSocialLoginPage = lazy(
 );
 
 const NotificationsPage = lazy(() => import('views/pages/notifications'));
-const NotificationSettingsPage = lazy(
-  () => import('views/pages/notification_settings'),
-);
+const NotificationSettingsPage = lazy(() => {
+  const knockInAppNotifications = useFlag('knockInAppNotifications');
+
+  if (knockInAppNotifications) {
+    return import('views/pages/NotificationSettings');
+  } else {
+    return import('views/pages/NotificationSettingsOld');
+  }
+});
```

Comment:
same as https://github.com/hicommonwealth/commonwealth/pull/7910/files#r1639881925

Reply:
please update this similar to how you did with `CommonDomainRoutes`

Reply:
@burtonator

Reply:
Fixed this but not sure why we have TWO routes here...